portfolio/src/main/webapp/script.js

```diff
@@ -16,12 +16,28 @@ document.addEventListener("DOMContentLoaded", function(){
   getComments();
 });
 
-// const selectElement = document.querySelector('#languageCode');
-// if (selectElement != null) {
-//   selectElement.addEventListener('change', (event) => {
-//     getComments();
-//   });
-// }
+function setSelections() {
+    document.myForm.languageCode.value = getQueryValue("languageCode");
+}
+
+function getQueryValue(key)
```

Comment:
Nit: for consistency, write open bracket `{` the same line as function declaration.